notification/test/notification/controllers/MainSpec.scala

```diff
@@ -60,6 +60,30 @@ class MainSpec(implicit ec: ExecutionEnv) extends PlaySpecification with Mockito
     }
   }
 
+  "adding a new notification topic" should {
+    "successfully add a new topic and send a notification to it" in new MainScope {
+      val newTopic = Topic(Breaking, "new-topic")
+      val topicsWithNew = validTopics :+ newTopic
+      val request = authenticatedRequest.withBody(breakingNewsNotification(topicsWithNew))
+      val response = main.pushTopics()(request)
+
+      status(response) should equalTo(CREATED)
+      pushSent must beSome.which(_.topic.toSet must beEqualTo(topicsWithNew.toSet))
+    }
+  }
+
+  "removing a new notification topic" should {
+    "successfully remove a topic and stop sending notifications to it" in new MainScope {
+      val topicToRemove = validTopics.head
```

Comment:
Small point, but is this variable being used anywhere? If not let's remove it

Reply:
ah yeah we're not using it anywhere I will remove it, thanks for spotting